services/addons/images/iss_health_check/iss_token.py

```diff
@@ -1,11 +1,31 @@
 from google.cloud import secretmanager
+import common.pgquery as pgquery
 import requests
 import os
 import json
 import uuid
 import logging
 
+# Get storage type from environment variable
+def get_storage_type():
+    """Get the storage type for the ISS SCMS API token
+    """
+    try :
+        os.environ["STORAGE_TYPE"]
+    except KeyError:
+        logging.error("STORAGE_TYPE environment variable not set, exiting")
+        exit(1)
+    
+    if os.environ["STORAGE_TYPE"] == "gcp":

```

Comment:
We can probably be nice here and do an equals ignore case

Reply:
This check is now case-insensitive. A couple more unit tests were added for this.